test/test_coretasks.py

```diff
@@ -35,7 +34,7 @@ def test_bot_mixed_modes(mockbot, ircfactory):
 
     .. seealso::
 
-        GitHub issue #1575.
+        GitHub issue #1575 (https://github.com/sopel-irc/sopel/pull/1575).
```

Comment:
Not linking the text `#1575` directly?

Reply:
Who will generate the HTML from the test file?

Reply:
Future-proofing? I mean, you've used Sphinx markup already with the `.. seealso::`, so… I guess I'm asking "why not?" 😹

Reply:
Heh, as discussed, worse case scenario: Sphinx catches the URL and display a link nonetheless. I say: don't bother adding too much noises - the `.. seealso::` helps a bit with structure, I can't say the same for a `#XXXX__` + `__ :: http...` stuff.